src/Helper.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <iostream>
+#include <fstream>
+#include <experimental/filesystem>
+namespace fs = std::experimental::filesystem;
+
+
+
+struct Helper
+{
+    static const std::string INPUT_FILES_DIRECTORY;
```

Comment:
Why a static field and not a class field that is filled in the constructor?

Reply:
Done